src/types.ts

```diff
@@ -1927,11 +1927,12 @@ type MergeTypedResponse<T> = T extends Promise<infer T2>
 //////                             /////
 ////////////////////////////////////////
 
-export type FormValue = string | File
+export type FormValue = string | File | Blob
```

Comment:
I think `File` is not necessary:

```ts
export type FormValue = string | Blob
```

Reply:
Thanks and that's right!
https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAZgSwDYFMYF4ZhQdxgMWRQAoBtAIgAtwRyBdAGhipoDooAPKcgSgG4AUKEiw4IAE4BbACIBDKLIxZcBCTPmzi-AWKlyFrCCijFykgJ6JU5JlZT8gA